src/internal/connector/graph/consts.go

```diff
@@ -25,3 +25,10 @@ const (
 	// given endpoint.
 	PreviousPathFileName = "previouspath"
 )
+
+// ---------------------------------------------------------------------------
+// Runtime Configuration
+// ---------------------------------------------------------------------------
+
+// URLItemFetchParallelism controls the worker pool size for streaming items.
+const URLItemFetchParallelism = 4
```

Comment:
this loses context about what service it's for. I assume OneDrive may have different parallelism needs than Exchange

Reply:
maybe missing a push? I still see the generic name from the previous commit

Reply:
heh, I expect people to be slow in response, so I'll say "it's done!" and then go work on the change.  😳